Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -496,6 +487,11 @@ theorem induction {p : Ordinal.{u} → Prop} (i : Ordinal.{u}) (h : ∀ j, (∀
     p i :=
   lt_wf.induction i h
 
+@[deprecated (since := "2024-10-09")]
```

Comment:
Let's name the lemma this is deprecated in favour of.

Reply:
This lemma would've been `Subsingleton.eq`, so the suggestion wouldn't have been very useful. I added `PrincipalSeg.eq` (matching `InitialSeg.eq`) and given that as the suggestion.

Reply:
Although... given that you can't just apply `PrincipalSeg.eq` directly, maybe it is useful to have this lemma. I'll undeprecate it but keep the golfed version.